pom.xml

```diff
@@ -23,7 +23,7 @@
   </licenses>
 
   <properties>
-    <bioformats.version>6.0.0</bioformats.version>
```

Comment:
Should be reverted. While the component version should be bumped to the next `SNAPSHOT`, we do not want it to have `SNAPSHOT` dependencies.